SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -500,6 +489,53 @@ void FLocalDeploymentManager::FinishLocalDeploymentShutDown()
 	bStoppingDeployment = false;
 }
 
+void FLocalDeploymentManager::WindowsRequestGracefulRuntimeShutdown()
+{
+	// Sends a process signal requesting the runtime shutdown
+
+	// Prepare runtime process name
+	FString RuntimeProcessName = RuntimePath;
```

Comment:
Should be able to do:
FString RuntimeProcessName = RuntimePath.Replace(TEXT("/"), TEXT("\\"));

Reply:
could even make it const

Reply:
changed!